utils/core-modules/contracts/modules/FeatureFlagModule.sol

```diff
@@ -20,9 +20,23 @@ contract FeatureFlagModule is IFeatureFlagModule {
         OwnableStorage.onlyOwner();
         FeatureFlag.load(feature).allowAll = allowAll;
 
+        if (allowAll) {
+            FeatureFlag.load(feature).denyAll = false;
+        }
+
         emit FeatureFlagAllowAllSet(feature, allowAll);
     }
 
+    /**
+     * @inheritdoc IFeatureFlagModule
+     */
+    function setFeatureFlagDenyAll(bytes32 feature, bool denyAll) external override {
+        OwnableStorage.onlyOwner();
+        FeatureFlag.load(feature).denyAll = denyAll;
+
+        emit FeatureFlagDenyAllSet(feature, denyAll);
+    }
+
```

Comment:
We also probably want to add a `getFeatureFlagAllowAll` fn.

Reply:
yeah lets add this too

Reply:
how I'm suppose to know all the features that has been added?

Reply:
This would just be a view function that returns the denyAll boolean for a particular feature, just like we do for the allowAll feature: `tureFlagAllowAll(bytes32 feature)`

Reply:
ah okay I thought you meant all existing features